src/boot/efi/pe.c

```diff
@@ -86,8 +86,7 @@ EFI_STATUS pe_memory_locate_sections(CHAR8 *base, CHAR8 **sections, UINTN *addrs
                 return EFI_LOAD_ERROR;
 
         /* PE32+ Subsystem type */
-        if (pe->FileHeader.Machine != PE_HEADER_MACHINE_X64 &&
-            pe->FileHeader.Machine != PE_HEADER_MACHINE_I386)
+        if (!IN_SET(pe->FileHeader.Machine, PE_HEADER_MACHINE_X64, PE_HEADER_MACHINE_I386))
```

Comment:
This line fails the autopkg test, as `src/boot/efi/pe.c` does not include `IN_SET` from `src/basic/macro.h`.

Reply:
yeah, src/boot/efi should really be excluded from all of this, it doesn't actually share much code with the rest of the tree, as it is EFI code.